src/pages/CreateAssignment/CreateAssignment.js

```diff
@@ -46,6 +43,13 @@ function CreateAssignment(props) {
       });
 
     if (existingAssignment) prePopulateAssignment(existingAssignment);
+    let originalDate = new Date();
+    // The following are set to 0 to allow for edge case where teacher wants to include today in lesson dates.
+    originalDate.setHours(0);
+    originalDate.setMinutes(0);
+    originalDate.setSeconds(0);
+    originalDate.setMilliseconds(0);
+    setLessonCreationDate(originalDate);
```

Comment:
also is there a way to use the Date constructor to set these all to 0?

Reply:
you can do `originalDate.setHours(0, 0, 0, 0)`- that sets hours, min, s, ms all to 0